ios/Exponent/Kernel/AppLoader/CachedResource/EXManifestResource.m

```diff
@@ -29,6 +30,10 @@ @interface EXManifestResource ()
 
 @implementation EXManifestResource
 
++ (void) initialize {
```

Comment:
```suggestion
+ (void)initialize {
```

Reply:
Using static methods like `initialize` and `load` are generally not a good practice. It's one of the reasons why React Native is moving away from macros as they expand to those methods to register the modules in runtime. The problem is that they are slowing down the startup time.